GroupBy(multiple groupers) (#9372)
pydata/xarray: doc/user-guide/groupby.rst

    ds.groupby("letters")["b"]

> Just like in pandas, creating a ``GroupBy`` object is cheap: it does not actually
> split the data until you access particular values.
> 
> To group by multiple variables, see the section on `Grouper Objects`_
> 

Author: ```suggestion
To group by multiple variables, see :ref:`this section <groupby.multiple>`.

```
